src/utils/string.hpp

```diff
@@ -301,6 +301,26 @@ inline std::vector<std::string> RSplit(const std::string_view src, const std::st
   return res;
 }
 
+/**
+ * Get a first part of a string by `delimiter` into a string.
+ */
+inline std::string_view GetFirstPartOfSplit(const std::string_view src, const char delimiter) {
```

Comment:
I like inline, makes sense here. How about returning std::string instead of string_view? Because then clients will always have to convert to string and it's again the same

Reply:
Yes, I thought about this. Will check performance, because we have one place DeserializePropertiesFromAuxiliaryStorages we pass string_view to function. But there should not be degradation. Will check